.github/workflows/playwright.yml

```diff
@@ -46,7 +46,9 @@ jobs:
           node-version: 18.12.1
 
       - name: Install dependencies
-        run: npm ci
+        run: |
+          npm ci
+          npx playwright install-deps webkit
```

Comment:
What dis?
I was trying to find a definitive list of browsers installed by default by `playwright install` but so far coming up empty. I was expecting it on https://playwright.dev/docs/browsers but no luck.

Although I did find the section on running in Chrome/Edge was pretty fascinating (if we wanted to get off Chromium and onto branded Chrome): https://playwright.dev/docs/browsers#google-chrome--microsoft-edge

Reply:
Actually, running `npx playwright install` locally [in an empty dir] either answered or confused my question...
```
npx playwright install
Need to install the following packages:
  playwright@1.38.0
Ok to proceed? (y) y

...

Downloading Chromium 117.0.5938.62 (playwright build v1080) from https://playwright.azureedge.net/builds/chromium/1080/chromium-mac-arm64.zip
128.1 Mb [====================] 100% 0.0s
Chromium 117.0.5938.62 (playwright build v1080) downloaded to /Users/pdehaan/Library/Caches/ms-playwright/chromium-1080

Downloading Firefox 117.0 (playwright build v1424) from https://playwright.azureedge.net/builds/firefox/1424/firefox-mac-13-arm64.zip
71.1 Mb [====================] 100% 0.0s
Firefox 117.0 (playwright build v1424) downloaded to /Users/pdehaan/Library/Caches/ms-playwright/firefox-1424

Downloading Webkit 17.0 (playwright build v1908) from https://playwright.azureedge.net/builds/webkit/1908/webkit-mac-13-arm64.zip
60.7 Mb [====================] 100% 0.0s
Webkit 17.0 (playwright build v1908) downloaded to /Users/pdehaan/Library/Caches/ms-playwright/webkit-1908
```

So it looks like a fresh <kbd>npx playwright install</kbd> installed:
* Chromium 117 (128 MB)
* Firefox 117 (71 MB)
* WebKit 17.0 (61 MB)

Was Playwright throwing errors if we didn't explicitly run <kbd>npx playwright install-deps webkit</kbd>?